lib/maze/client/appium/base_client.rb

```diff
@@ -23,11 +23,16 @@ def start_session
                                  Maze.driver.session_capabilities['appPackage']
                                when 'ios'
                                  unless app_id = Maze.driver.session_capabilities['CFBundleIdentifier']
-                                    app_id = Maze.driver.session_capabilities['bundleID']
+                                    app_id = Maze.driver.session_capabilities['bundleId']
                                  end
                                  app_id
                                end
 
+          if Maze.driver.app_id.nil?
+            $logger.error "Failed to determine app id."
+            $logger.debug "Capabilities: #{Maze.driver.session_capabilities.inspect}"
```

Comment:
@twometresteve not sure if we want this, I thought that outputting what capabilities are set when we've got debug enabled might help 🤷🏼

Reply:
Logging as debug seems like a good idea to me, but I would just change it as follows to avoid confusion with the capabilities that we set.
```suggestion
            $logger.debug "session_capabilities: #{Maze.driver.session_capabilities.inspect}"
```